management/terraform_suggested_version.rego

```diff
@@ -0,0 +1,14 @@
+package terraform
+
+import input.tfplan as tfplan
+
+suggested_version = "1.3.5"
+
+# Matches that minimum version is met
+suggested["Suggested version"] {
+	version := tfplan.terraform_version
+
+	version < suggested_version
+
+	reason := sprintf("Terraform version %s will soon be deprecated. Please migrate to version %s", [version, suggested_version])
+}
```

Comment:
This is the same as `terraform_min_version`, just with another deny message. I'd assume one example is enough.

Reply:
Simplified it to one example.